### ISSUE TITLE
LDES Server crashed with local build 

### ISSUE THREAD

**Issue by xdxxxdx** [Informatievlaanderen/VSDS-LDESServer4J#512]
**Prerequisites:**
Commit tags (sprint 24):
VSDS-LDESServer4J: 1f248a8f9303a5c92c166e3d50bdffe98b018d13.

**To Reproduce:**
1. Go to the working directory under `/VSDS-LDESServer4J`
2. Run `docker-compose up`

**Expected behavior**
LDES server is built and started with the local configuration `- ./docker-compose/config.local.env` as described at https://github.com/Informatievlaanderen/VSDS-LDESServer4J/blob/main/compose.yml

**Current result**
LDES Server crashed.
Log:

```
2023-04-14 11:47:20    ____  _        _             ____                            ____
2023-04-14 11:47:20   / / / | |    __| | ___  ___  / ___|  ___ _ ____   _____ _ __  \ \ \
2023-04-14 11:47:20  / / /  | |   / _` |/ _ \/ __| \___ \ / _ \ '__\ \ / / _ \ '__|  \ \ \
2023-04-14 11:47:20  \ \ \  | |__| (_| |  __/\__ \  ___) |  __/ |   \ V /  __/ |     / / /
2023-04-14 11:47:20   \_\_\ |_____\__,_|\___||___/ |____/ \___|_|    \_/ \___|_|    /_/_/
2023-04-14 11:47:20 
2023-04-14 11:47:20 Version 
2023-04-14 11:47:20 Powered by Spring Boot 3.0.5
2023-04-14 11:47:20 
2023-04-14 11:47:20 2023-04-14T09:47:20.413Z  INFO 1 --- [           main] b.v.i.ldes.server.Application            : Starting Application using Java 18-ea with PID 1 (/ldes-server-application.jar started by ldes in /)
2023-04-14 11:47:20 2023-04-14T09:47:20.417Z  INFO 1 --- [           main] b.v.i.ldes.server.Application            : No active profile set, falling back to 1 default profile: "default"
2023-04-14 11:47:24 2023-04-14T09:47:24.585Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data MongoDB repositories in DEFAULT mode.
2023-04-14 11:47:25 2023-04-14T09:47:25.530Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 938 ms. Found 4 MongoDB repository interfaces.
2023-04-14 11:47:26 2023-04-14T09:47:26.202Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2023-04-14 11:47:26 2023-04-14T09:47:26.216Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-04-14 11:47:26 2023-04-14T09:47:26.216Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.7]
2023-04-14 11:47:26 2023-04-14T09:47:26.328Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-04-14 11:47:26 2023-04-14T09:47:26.334Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 5859 ms
2023-04-14 11:47:27 Fri Apr 14 09:47:27 UTC 2023 Thread[main,5,main] java.io.FileNotFoundException: derby.log (Permission denied)
2023-04-14 11:47:27 ----------------------------------------------------------------
2023-04-14 11:47:27 Fri Apr 14 09:47:27 UTC 2023:
2023-04-14 11:47:27 Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a816c00e-0187-7f29-9fc6-000009ee8680 
2023-04-14 11:47:27 on database directory /tmp/Databases/SpatialMetadata with class loader org.springframework.boot.loader.LaunchedURLClassLoader@4783da3f 
2023-04-14 11:47:27 Loaded from file:/lib/ldes-fragmentisers-geospatial-jar-with-dependencies.jar
2023-04-14 11:47:27 java.vendor=Oracle Corporation
2023-04-14 11:47:27 java.runtime.version=18-ea+35-2085
2023-04-14 11:47:27 user.dir=/
2023-04-14 11:47:27 os.name=Linux
2023-04-14 11:47:27 os.arch=amd64
2023-04-14 11:47:27 os.version=5.15.90.1-microsoft-standard-WSL2
2023-04-14 11:47:27 derby.system.home=null
2023-04-14 11:47:27 Database Class Loader started - derby.database.classpath=''
2023-04-14 11:47:28 2023-04-14T09:47:28.784Z  INFO 1 --- [           main] org.apache.sis.referencing.factory       : Creating EPSG schema in the “jdbc:derby:/tmp/Databases/SpatialMetadata” database.
2023-04-14 11:47:47 2023-04-14T09:47:47.619Z  INFO 1 --- [           main] org.mongodb.driver.client                : MongoClient with metadata {"driver": {"name": "mongo-java-driver|sync|spring-boot", "version": "4.8.2"}, "os": {"type": "Linux", "name": "Linux", "architecture": "amd64", "version": "5.15.90.1-microsoft-standard-WSL2"}, "platform": "Java/Oracle Corporation/18-ea+35-2085"} created with settings MongoClientSettings{readPreference=primary, writeConcern=WriteConcern{w=null, wTimeout=null ms, journal=null}, retryWrites=true, retryReads=true, readConcern=ReadConcern{level=null}, credential=null, streamFactoryFactory=null, commandListeners=[org.springframework.data.mongodb.observability.MongoObservationCommandListener@2a0a178f, io.micrometer.core.instrument.binder.mongodb.MongoMetricsCommandListener@5f9cb96f], codecRegistry=ProvidersCodecRegistry{codecProviders=[ValueCodecProvider{}, BsonValueCodecProvider{}, DBRefCodecProvider{}, DBObjectCodecProvider{}, DocumentCodecProvider{}, CollectionCodecProvider{}, IterableCodecProvider{}, MapCodecProvider{}, GeoJsonCodecProvider{}, GridFSFileCodecProvider{}, Jsr310CodecProvider{}, JsonObjectCodecProvider{}, BsonCodecProvider{}, EnumCodecProvider{}, com.mongodb.Jep395RecordCodecProvider@55a7f0af]}, clusterSettings={hosts=[ldes.mongodb:27017], srvServiceName=mongodb, mode=SINGLE, requiredClusterType=UNKNOWN, requiredReplicaSetName='null', serverSelector='null', clusterListeners='[]', serverSelectionTimeout='30000 ms', localThreshold='30000 ms'}, socketSettings=SocketSettings{connectTimeoutMS=10000, readTimeoutMS=0, receiveBufferSize=0, sendBufferSize=0}, heartbeatSocketSettings=SocketSettings{connectTimeoutMS=10000, readTimeoutMS=10000, receiveBufferSize=0, sendBufferSize=0}, connectionPoolSettings=ConnectionPoolSettings{maxSize=100, minSize=0, maxWaitTimeMS=120000, maxConnectionLifeTimeMS=0, maxConnectionIdleTimeMS=0, maintenanceInitialDelayMS=0, maintenanceFrequencyMS=60000, connectionPoolListeners=[io.micrometer.core.instrument.binder.mongodb.MongoMetricsConnectionPoolListener@1633962a], maxConnecting=2}, serverSettings=ServerSettings{heartbeatFrequencyMS=10000, minHeartbeatFrequencyMS=500, serverListeners='[]', serverMonitorListeners='[]'}, sslSettings=SslSettings{enabled=false, invalidHostNameAllowed=false, context=null}, applicationName='null', compressorList=[], uuidRepresentation=JAVA_LEGACY, serverApi=null, autoEncryptionSettings=null, contextProvider=DefaultSynchronousContextProvider[observationRegistry=io.micrometer.observation.SimpleObservationRegistry@2bfaba70]}
2023-04-14 11:47:47 2023-04-14T09:47:47.670Z  INFO 1 --- [s.mongodb:27017] org.mongodb.driver.cluster               : Monitor thread successfully connected to server with description ServerDescription{address=ldes.mongodb:27017, type=STANDALONE, state=CONNECTED, ok=true, minWireVersion=0, maxWireVersion=17, maxDocumentSize=16777216, logicalSessionTimeoutMinutes=30, roundTripTimeNanos=47043409}
2023-04-14 11:47:48 2023-04-14T09:47:48.229Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.fragment.entity.LdesFragmentEntity
2023-04-14 11:47:48 2023-04-14T09:47:48.268Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset1.entities.LdesFragmentEntityV2
2023-04-14 11:47:48 2023-04-14T09:47:48.282Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset1.entities.LdesMemberEntityV1
2023-04-14 11:47:48 2023-04-14T09:47:48.287Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset1.entities.LdesFragmentEntityV1
2023-04-14 11:47:48 2023-04-14T09:47:48.296Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.member.entity.LdesMemberEntity
2023-04-14 11:47:48 2023-04-14T09:47:48.301Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset1.entities.LdesMemberEntityV2
2023-04-14 11:47:48 2023-04-14T09:47:48.306Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.ldesconfig.entity.LdesConfigModelEntity
2023-04-14 11:47:48 2023-04-14T09:47:48.308Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset2.entities.LdesMemberEntityV3
2023-04-14 11:47:48 2023-04-14T09:47:48.316Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.snapshot.entity.SnapshotEntity
2023-04-14 11:47:48 2023-04-14T09:47:48.320Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset2.entities.LdesMemberEntityV2
2023-04-14 11:47:48 2023-04-14T09:47:48.866Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.fragment.repository.LdesFragmentEntityRepository
2023-04-14 11:47:49 2023-04-14T09:47:49.042Z  WARN 1 --- [           main] edFragmentationStrategyAutoConfiguration : Using deprecated timebased fragmentation. For more information, refer to https://github.com/Informatievlaanderen/VSDS-LDESServer4J/blob/main/ldes-fragmentisers/ldes-fragmentisers-timebased/README.MD
2023-04-14 11:47:49 2023-04-14T09:47:49.066Z  INFO 1 --- [           main] .v.i.l.s.f.m.DirectFragmentationMediator : Server has been configured to NOT use a queue for fragmentation
2023-04-14 11:47:49 2023-04-14T09:47:49.108Z  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.ldesconfig.repository.LdesConfigEntityRepository
2023-04-14 11:47:49 2023-04-14T09:47:49.441Z  INFO 1 --- [           main] i.m.r.core.builder.RunnerBuilderBase     : Mongock runner COMMUNITY version[5.2.4]
2023-04-14 11:47:49 2023-04-14T09:47:49.442Z  INFO 1 --- [           main] i.m.r.core.builder.RunnerBuilderBase     : Running Mongock with NO metadata
2023-04-14 11:47:49 2023-04-14T09:47:49.442Z  WARN 1 --- [           main] i.m.r.core.builder.RunnerBuilderBase     : Property transaction-enabled not provided. It will become true as default in next versions. Set explicit value to false in case transaction are not desired.
2023-04-14 11:47:49 2023-04-14T09:47:49.442Z  WARN 1 --- [           main] i.m.r.core.builder.RunnerBuilderBase     : Property transaction-enabled not provided and is unknown if driver is transactionable. BY DEFAULT MONGOCK WILL RUN IN NO-TRANSACTION MODE.
2023-04-14 11:47:49 2023-04-14T09:47:49.449Z  WARN 1 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'applicationRunner' defined in class path resource [io/mongock/runner/springboot/config/MongockContext.class]: Failed to instantiate [io.mongock.runner.springboot.base.MongockApplicationRunner]: Factory method 'applicationRunner' threw exception with message: Scan package for changeLogs is not set: use appropriate setter
2023-04-14 11:47:49 2023-04-14T09:47:49.472Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-04-14 11:47:49 2023-04-14T09:47:49.495Z  INFO 1 --- [           main] .s.b.a.l.ConditionEvaluationReportLogger : 
2023-04-14 11:47:49 
2023-04-14 11:47:49 Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2023-04-14 11:47:49 2023-04-14T09:47:49.524Z ERROR 1 --- [           main] o.s.boot.SpringApplication               : Application run failed
2023-04-14 11:47:49 
2023-04-14 11:47:49 org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'applicationRunner' defined in class path resource [io/mongock/runner/springboot/config/MongockContext.class]: Failed to instantiate [io.mongock.runner.springboot.base.MongockApplicationRunner]: Factory method 'applicationRunner' threw exception with message: Scan package for changeLogs is not set: use appropriate setter
2023-04-14 11:47:49 at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:657) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:645) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1332) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1162) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:560) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:520) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:326) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:324) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:973) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:917) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:584) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:3.0.5]
2023-04-14 11:47:49 at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:732) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:3.0.5]
2023-04-14 11:47:49 at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:434) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:3.0.5]
2023-04-14 11:47:49 at org.springframework.boot.SpringApplication.run(SpringApplication.java:310) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:3.0.5]
2023-04-14 11:47:49 at org.springframework.boot.SpringApplication.run(SpringApplication.java:1304) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:3.0.5]
2023-04-14 11:47:49 at org.springframework.boot.SpringApplication.run(SpringApplication.java:1293) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:3.0.5]
2023-04-14 11:47:49 at be.vlaanderen.informatievlaanderen.ldes.server.Application.main(Application.java:9) ~[classes!/:na]
2023-04-14 11:47:49 at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[na:na]
2023-04-14 11:47:49 at java.base/java.lang.reflect.Method.invoke(Method.java:577) ~[na:na]
2023-04-14 11:47:49 at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49) ~[ldes-server-application.jar:na]
2023-04-14 11:47:49 at org.springframework.boot.loader.Launcher.launch(Launcher.java:95) ~[ldes-server-application.jar:na]
2023-04-14 11:47:49 at org.springframework.boot.loader.Launcher.launch(Launcher.java:58) ~[ldes-server-application.jar:na]
2023-04-14 11:47:49 at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:466) ~[ldes-server-application.jar:na]
2023-04-14 11:47:49 Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.mongock.runner.springboot.base.MongockApplicationRunner]: Factory method 'applicationRunner' threw exception with message: Scan package for changeLogs is not set: use appropriate setter
2023-04-14 11:47:49 at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:171) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:653) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 ... 25 common frames omitted
2023-04-14 11:47:49 Caused by: io.mongock.api.exception.MongockException: Scan package for changeLogs is not set: use appropriate setter
2023-04-14 11:47:49 at io.mongock.runner.core.executor.ExecutorBuilderBase.validateScanPackage(ExecutorBuilderBase.java:131) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at io.mongock.runner.core.executor.ExecutorBuilderDefault.getExecutorByOperation(ExecutorBuilderDefault.java:28) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at io.mongock.runner.core.executor.ExecutorBuilderBase.buildOperationExecutor(ExecutorBuilderBase.java:87) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at io.mongock.runner.core.builder.RunnerBuilderBase.buildOperationExecutor(RunnerBuilderBase.java:239) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at io.mongock.runner.core.builder.RunnerBuilderBase.buildRunner(RunnerBuilderBase.java:146) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at io.mongock.runner.core.builder.RunnerBuilderBase.buildRunner(RunnerBuilderBase.java:131) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at io.mongock.runner.core.builder.RunnerBuilderBase.buildRunner(RunnerBuilderBase.java:126) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at io.mongock.runner.springboot.base.builder.SpringbootBuilderBase.buildApplicationRunner(SpringbootBuilderBase.java:103) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at io.mongock.runner.springboot.base.config.MongockContextBase.applicationRunner(MongockContextBase.java:27) ~[mongock-changeset-1-jar-with-dependencies.jar:na]
2023-04-14 11:47:49 at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[na:na]
2023-04-14 11:47:49 at java.base/java.lang.reflect.Method.invoke(Method.java:577) ~[na:na]
2023-04-14 11:47:49 at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:139) ~[ldes-fragmentisers-geospatial-jar-with-dependencies.jar:6.0.7]
2023-04-14 11:47:49 ... 26 common frames omitted
2023-04-14 11:47:49 
2023-04-14 11:47:49 ----------------------------------------------------------------
2023-04-14 11:47:49 Fri Apr 14 09:47:49 UTC 2023:
2023-04-14 11:47:49 Shutting down instance a816c00e-0187-7f29-9fc6-000009ee8680 on database directory /tmp/Databases/SpatialMetadata with class loader org.springframework.boot.loader.LaunchedURLClassLoader@4783da3f 
```
**Expected result**
LDES Server should not crash

**Comment by WLefever-Cegeka** [comment]
Hi @xdxxxdx ,
Can you verify that the problem is solved in #513 ?

**Comment by xdxxxdx** [comment]
Validated on the commit:  VSDS-LDESServer4J: e1c62110b5926f151f75b3daf26254c83a87c1a6
